engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -1,8 +1,7 @@
-// Copyright 2020 The Terasology Foundation
+// Copyright 2021 The Terasology Foundation
```

Comment:
Not sure where to put this: when we previously migrated `Camera` we just replace `math.geom.Vector3i` with `org.joml.Vector3i`. Due to this, we expose the internal `position` is a way that it can (accidentally) be modified.

I think we should do another pass over this class to fix this.